cmd/internal/flowmode/cmd_run.go

```diff
@@ -334,6 +339,50 @@ func (fr *flowRun) Run(configFile string) error {
 		return err
 	}
 
+	// Reload on file change
+	if fr.configAutoReload {
```

Comment:
I wonder if this code block is getting to long and deeply nested and if we should move it to its own func.

Reply:
I agree, especially if polling is going to be added. That'll come in with the change